docs/api_guide/tutorial_pynapple_wavelets.py

```diff
@@ -0,0 +1,489 @@
+# -*- coding: utf-8 -*-
+"""
+Wavelet Transform
+=================
+
+This tutorial covers the use of `nap.compute_wavelet_transform` to do continuous wavelet transform. By default, pynapple uses Morlet wavelets.
```

Comment:
Add that wavelet are a great tool to capture changes of spectral characteristics of a signal over time. basically, why it is worth using wavelet transforms